schema.py

```
@@ -93,11 +99,11 @@ def generate(output_dir):
     ensure_dir(output_dir)
 
     for schema_name in ["pg_catalog", "information_schema"]:
-        print(f"processing schema: {schema_name}")
+        logger.info("processing schema: %s", schema_name)
         objects = fetch_objects(conn, schema_name)
```

Pylint (3.3.7)

[error] Possibly using variable 'conn' before assignment

(E0606)